jOOQ/src/main/java/org/jooq/impl/DSL.java

```diff
@@ -7852,6 +7853,18 @@ public static DropIndexOnStep dropIndexIfExists(Index index) {
         return new QuantifiedSelectImpl<Record1<T>>(Quantifier.ANY, array);
     }
 
+    // -------------------------------------------------------------------------
+    // XXX Access control
+    // -------------------------------------------------------------------------
+
+    public static User user(String name) {
+        return new UserImpl(name);
+    }
+
+    public static User user(Name name) {
+        return new UserImpl(name.toString());
```

Comment:
This should be the main constructor. The one taking the `String` argument should be the convenience constructor, calling `user(name(name))`